tests/ord-service/tests/api_test.go

```diff
@@ -166,321 +120,374 @@ func TestORDService(t *testing.T) {
 		makeRequestWithStatusExpect(t, unsecuredHttpClient, testConfig.ORDServiceURL+"/$metadata?$format=json", http.StatusUnauthorized)
 	})
 
-	t.Run("Requesting entities without specifying response format falls back to configured default response type when Accept header allows everything", func(t *testing.T) {
-		makeRequestWithHeaders(t, httpClient, testConfig.ORDServiceURL+"/bundles", map[string][]string{acceptHeader: {"*/*"}})
+	t.Run("400 when requests to ORD Service do not have tenant header", func(t *testing.T) {
+		makeRequestWithStatusExpect(t, httpClient, testConfig.ORDServiceURL+"/bundles?$format=json", http.StatusBadRequest)
 	})
 
-	t.Run("Requesting entities without specifying response format falls back to response type specified by Accept header when it provides a specific type", func(t *testing.T) {
-		makeRequestWithHeaders(t, httpClient, testConfig.ORDServiceURL+"/bundles", map[string][]string{acceptHeader: {"application/json"}})
+	t.Run("400 when requests to ORD Service have wrong tenant header", func(t *testing.T) {
+		makeRequestWithHeadersAndStatusExpect(t, httpClient, testConfig.ORDServiceURL+"/bundles?$format=json", map[string][]string{tenantHeader: {"wrong-tenant"}}, http.StatusBadRequest)
 	})
 
-	t.Run("Requesting Bundles returns them as expected", func(t *testing.T) {
-		respBody := makeRequest(t, httpClient, testConfig.ORDServiceURL+"/bundles?$format=json")
-
-		require.Equal(t, len(appInput.Bundles), len(gjson.Get(respBody, "value").Array()))
-		require.Equal(t, appInput.Bundles[0].Name, gjson.Get(respBody, "value.0.title").String())
-		require.Equal(t, *appInput.Bundles[0].Description, gjson.Get(respBody, "value.0.description").String())
+	t.Run("400 when requests to ORD Service api specification do not have tenant header", func(t *testing.T) {
```

Comment:
All of the specification tests do not really test only the tenancy since you use dummy ID and this will fail anyways either if the tenant is right or wrong